### PR TITLE
SUS-379 Make chat use data from consul instead of ChatConfig.json

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/SUS-379

**DO NOT MERGE!**
until this ticket https://wikia-inc.atlassian.net/browse/OPS-8565 will be solved.

This PR is moving chat service discovery to consul instead of using ChatConfig.json.
Config PR:https://github.com/Wikia/config/pull/1738

### CODE REVIEW

File: lib/Wikia/src/Consul/Client.php [hunk 1]
```diff
@@ -25,32 +25,39 @@ class Client {
...
-	 * @return array list of IP addresses
+	 * @return array list of IP addresses with ports ie. 127.0.0.1:1234
 	 */
 	function getNodes( $service, $tag ) {
-		$resp = $this->api->service( $service, [ 'tag' => $tag, 'passing' => true ] )->json();
+		$resp = $this->api->service( $service, [
+			'tag' => $tag,
+			'passing' => true,
+			// TODO: temporary change for devbox testing
+			'dc' => 'sjc'
```
Comment (wladekb) [inline lib/Wikia/src/Consul/Client.php]: This should disappear before PR is merged.

File: lib/Wikia/src/Consul/Client.php [hunk 2]
```diff
@@ -25,32 +25,39 @@ class Client {
...
+		] )->json();
 
 		$nodes = array_map(
-			function( $item ) {
-				return $item[ 'Node' ][ 'Address' ];
+			function ( $item ) {
+				return $item[ 'Node' ][ 'Address' ] . ':' . $item[ 'Service' ][ 'Port' ];
```
Comment (wladekb) [inline lib/Wikia/src/Consul/Client.php:52]: Is this change compatible with all other uses among our codebase?

Comment (dianafa) [inline lib/Wikia/src/Consul/Client.php:52]: yes

File: lib/Wikia/src/Consul/Client.php (hunk 1 above)
Comment (dianafa) [inline lib/Wikia/src/Consul/Client.php]: yep, fixing now

**Comment by dianafa** [comment]
this will be CR, tested and merged as a part of : https://github.com/Wikia/app/pull/10243/files

**Pull request closed without merging**